sw/device/tests/rv_core_ibex_epmp_test.c

```diff
@@ -0,0 +1,336 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/arch/device.h"
+#include "sw/device/lib/base/csr.h"
+#include "sw/device/lib/dif/dif_uart.h"
+#include "sw/device/lib/runtime/ibex.h"
+#include "sw/device/lib/runtime/log.h"
+#include "sw/device/lib/runtime/pmp.h"
+#include "sw/device/lib/runtime/print.h"
+#include "sw/device/lib/testing/pinmux_testutils.h"
+#include "sw/device/lib/testing/test_framework/check.h"
+#include "sw/device/lib/testing/test_framework/ottf_test_config.h"
+#include "sw/device/lib/testing/test_framework/status.h"
+#include "sw/device/silicon_creator/lib/dbg_print.h"
+#include "sw/device/silicon_creator/lib/epmp_defs.h"
+
+#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
+
+OTTF_DEFINE_TEST_CONFIG();
+
+/**
+ * Labels
+ */
+// The end of the rodata region.
+// This is set in the linker script.
+extern const char __rodata_end[];
+// The start and the end of the user's test function.
+extern const char i_am_become_user[];
+extern const char kIAmBecomeUserEnd[];
+// Expected U Mode Instruction Access Fault Return Address
```

Comment:
The comments are inconsistent throughout the file.
```suggestion
// Expected U Mode Instruction Access Fault Return Address.
```
It's ok to do in the follow-up PR as this passed CI.